.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+// Coloque su configuración en este archivo para sobrescribir la configuración predeterminada y de usuario.
```

Comment:
I think is a good idea to keep it, we should have as well atom settings. If somebody arrives to the project he will get the right settings for free